libs/bits/bit_array.go

```diff
@@ -256,7 +256,7 @@ func (bA *BitArray) PickRandom() (int, bool) {
 		return 0, false
 	}
 
-	return trueIndices[tmrand.Intn(len(trueIndices))], true
+	return trueIndices[mrand.Intn(len(trueIndices))], true
```

Comment:
Note to self: update above comment `// It uses the global randomness in `random.go` to get this index.`